recipes/message-ix/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.0-prerelease" %}
+{% set sha256 = "b97297174dc7a3da925b8ba45affa8b7b1a24dd2b9e535876cfd180ffc863d92" %}
+
+package:
+  name: message-ix
+  version: {{ version }}
+
+source:
+  url: https://github.com/iiasa/message_ix/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Can this package be noarch?

Reply:
It should be @mariusvniekerk - added in cf0df8e, should pass CI